nautobot/core/apps/__init__.py

```diff
@@ -513,13 +512,15 @@ def fixed_fields(self):
     args = []
     kwargs = {}
 
-    def __init__(self, link, name, args=[], kwargs={}, permissions=None, buttons=(), weight=1000):
+    def __init__(self, link, name, args=None, kwargs=None, permissions=None, buttons=(), weight=1000):
```

Comment:
```suggestion
    def __init__(self, link, name, args=None, kwargs=None, permissions=None, buttons=(), weight=1000):
        args = args | []
        kwargs = kwargs | {}
```

Reply:
Is this even needed or can we just leave them as None?

If needed, I'd prefer doing this inline below as just:

```python
self.args = args or []
self.kwargs = kwargs or {}
```

Reply:
I dont think so? [reverse()'s args and kwargs](https://docs.djangoproject.com/en/4.1/ref/urlresolvers/) are default to `None`. so it wont cause any problems if `args = None` or `args = []`